images/scripts/opensuse-tumbleweed.setup

```diff
@@ -89,7 +92,7 @@ printf '[main]\nno-auto-default=*\n' > /etc/NetworkManager/conf.d/noauto.conf
 echo "127.0.1.1 $(hostname)" >> /etc/hosts
 
 if ! getent passwd admin >/dev/null; then
-    useradd -c Administrator admin
+    useradd -c Administrator -G wheel admin
```

Comment:
Curious -- I booted the current main image, and I *can* login as `admin@` and *can* already sudo, and the admin user already is in `wheel`. So this probably means that the cloud image has an admin user by default? But then, how did it get its "foobar" password? Something doesn't add up here..

Reply:
That's interesting, I assumed based on the original code that admin is not created by default. But if it is and it's in wheel already, we can get rid of the whole if statement. Thanks for pointing this out!

Reply:
Indeed the [most recent build log](https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-opensuse-tumbleweed-be5a86cc-20240323-053844/log.html) shows:
```
+ getent passwd admin
+ '[' opensuse-tumbleweed '!=' opensuse-tumbleweed ']'
```
I.e. admin is already present. You'd still need the `chpasswd` that sets the password, but I figure that's already taken care of by cloud-init. So indeed you can drop this  whole paragraph.